tst/testinstall/methwhy.tst

```diff
@@ -1,8 +1,10 @@
-gap> START_TEST("methwy.tst");
+gap> START_TEST("methwhy.tst");
 
 #
 gap> ApplicableMethod();
-Error, usage: ApplicableMethod(<opr>,<arglist>[,<verbosity>[,<nr>]])
+#I  ApplicableMethod requires at least two arguments
+#I  usage: ApplicableMethod(<opr>,<arglist>[,<verbosity>[,<nr>]])
```

Comment:
Hmm, so this function now handles argument validation differently from (almost?) any other - why?

Reply:
I'm not entirely clear what you mean here. I understand returning fail when there is no method to pick, but it seems strange to return fail when the arguments just weren't valid (as then someone might get fail, because they mistyped the arguments).

Reply:
I agree with @ChrisJefferson